### ISSUE TITLE
Lab runs against the latest dependencies

### ISSUE THREAD

**Issue by hongdai** [dotnet/wcf#772]
We have recently changed the lab run to run against minimum dependencies. We need to be able to run against the latest dependencies as well as the minimum dependencies.

Update:
To solve this problem, we will need below items:

- [ ] Modify build scripts to enable latest dependencies configuration.
- [ ] Have a Jekins job to run this configuration routinely.

**Comment by iamjasonp** [comment]
By "routinely", are we talking about "all the time" or "in addition to" running against minimum dependencies? 

I'm thinking maybe keep most testing running against minimum dependencies, and additionally having a job that runs against latest dependencies - once upon each commit and also (say) once a day.

**Comment by zhenlan** [comment]
Running against latest should be "in addition to" running against minimum. The latter is more important.

Regarding the frequency, @iamjasonp, is there a way to tell CI to run it once a day? Are you concerned about the extra time this takes to run against every PR?

**Comment by mconnew** [comment]
I am against having it run automatically for a PR as purpose is verify that none of our dependencies have introduced a regression. A failure caused by a dependency regression when verifying a PR will only lead to confusion.

**Comment by iamjasonp** [comment]
We can set it to run on any frequency we wish (e.g., hourly/daily/weekly, etc.) 

One option is that it's possible for us to run not as part of a PR (i.e., this wouldn't be reported when verifying the PR) but only after each *merge*. 

The downside, of course, is that we probably aren't going to be looking at these ancillary runs all too often (maybe only TestCop will), and the PR committer will not get a visual indicator that they've introduced something breaking.  

I'm not concerned about the extra time it runs against every PR; we'd probably queue this on yet another machine in the pool

**Comment by zhenlan** [comment]
I think @mconnew raised a good point - we should not randomize our daily development with churns from other dependencies that are also in active development. We probably can run this once a week at the most. If we cannot have this scheduled automatically, a "@dotnet-bot test against latest dependencies please" will be nice to have.
